### DISCUSSION TITLE
Snowflake Datatype

### DISCUSSION THREAD

**Discussion by syncblaze** [mccoderpy/discord.py-message-components#48] [Ideas]
How about implementing a Datatype for Snowflakes, like in [Hikari](https://github.com/hikari-py/hikari/blob/master/hikari/snowflakes.py) or in [d.py 2.0](https://github.com/Rapptz/discord.py/blob/master/discord/types/snowflake.py)?

In my opinion is the way its done way better in hikari then in d.py 2.0

**Comment by mccoderpy** [comment]
> How about implementing a Datatype for Snowflakes, like in [Hikari](https://github.com/hikari-py/hikari/blob/master/hikari/snowflakes.py) or in [d.py 2.0](https://github.com/Rapptz/discord.py/blob/master/discord/types/snowflake.py)?
> 
> In my opinion is the way its done way better in hikari then in d.py 2.0

Basically a good idea. Could be added to the ToDo for the v2 typehinting
